### ISSUE TITLE
Contrasto insufficiente del testo in dark mode

### ISSUE THREAD

**Issue by martalucivero** [italia/graph-italia#114]
### Esiste già una discussione sul tema che ti interessa, o su un tema simile?

- [x] Ho verificato e non esiste

### Cosa

Il contrasto cromatico delle scritte relative ai testi "livello 1" e "livello 3" presenti in homepage non è sufficiente rispetto allo sfondo.
Il rapporto di contrasto non raggiunge il valore minimo previsto dalle WCAG per il testo.

<img width="1321" height="655" alt="Image" src="https://github.com/user-attachments/assets/b326f100-d747-4b9e-b3fc-4b605705fdd6" />

### Perché

WCAG 1.4.3 – Contrast (Minimum) (Livello AA): il testo deve avere un rapporto di contrasto di almeno 4,5:1, oppure 3:1 nel caso di testo grande.
